.travis.yml

```diff
@@ -48,6 +50,11 @@ before_install:
     - if [[ $deps != skip ]]; then composer self-update; fi;
     - if [[ $deps != skip ]]; then ./phpunit install; fi;
     - export PHPUNIT=$(readlink -f ./phpunit)
+    - mkdir /tmp/slapd
+    - slapd -f src/Symfony/Component/Ldap/Tests/Fixtures/conf/slapd.conf -h ldap://localhost:3389 &
+    - sleep 3
```

Comment:
slapd is run in the background. Also, this ensures that the daemon is started before the test are run. We could remove this, but tests may be flaky.